pkg/auth/handler/webapp/authflow_enter_password.go

```diff
@@ -38,6 +38,7 @@ type AuthflowEnterPasswordViewModel struct {
 	PasswordManagerUsername string
 	ForgotPasswordInputType string
 	ForgotPasswordLoginID   string
+	CurrentXStep            string
```

Comment:
CurrentXStep is unused. Please remove it.

Reply:
@tung2744 In case you missed this.

Reply:
@louischan-oursky  Removed, thanks!